server/test/integration/serverTest.js

```diff
@@ -81,9 +104,27 @@ describe('Brave Sensor server', () => {
       })
     })
 
-    describe('for a value STILLNESS request', () => {
+    describe('for an otherwise valid STILLNESS request with an incorrect API key', () => {
       beforeEach(async () => {
-        await firmwareAlert(chai, server, radar_coreID, SENSOR_EVENT.STILLNESS)
+        await firmwareAlert(chai, server, radar_coreID, SENSOR_EVENT.STILLNESS, badpassword)
+      })
+
+      it('should return 200', () => {
+        expect(this.response).to.have.status(200)
+      })
+
+      it('should log the error', () => {
+        expect(helpers.logError).to.be.calledWithExactly(`Access not allowed: incorrect key ${badpassword}`)
+      })
+
+      it('should not start the alert session since access should not have been given', () => {
```

Comment:
This is great. I'm glad that you thought to check this.

Reply:
thank you :)